src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  const getSum = (x) => {
+    return function(y) {
```

Comment:
use one same type of functions, in this case, it even would be better to change these two functions to arrow functions